src/urllib3/response.py

```diff
@@ -455,6 +457,13 @@ def data(self):
         if self._fp:
             return self.read(cache_content=True)
 
+    def json(self) -> Any:
```

Comment:
Because this function relies on `data` we can implement under `BaseHTTPResponse`

Reply:
Sure